nat-instance.tf

```diff
@@ -71,7 +75,7 @@ resource "aws_instance" "nat_instance" {
   ami                    = join("", data.aws_ami.nat_instance.*.id)
   instance_type          = var.nat_instance_type
   subnet_id              = element(aws_subnet.public.*.id, count.index)
-  vpc_security_group_ids = [aws_security_group.nat_instance[0].id]
+  vpc_security_group_ids = compact(concat(module.nat_instance_security_group.*.id, var.security_groups))
```

Comment:
@SweetOps Bridgecrew suggests using a network interface with a configured vpc but this ec2 instance has subnets provided which puts this ec2 in a vpc so I believe this rule should be ignored so it can pass the tests.